src/settings.rs

```diff
@@ -21,6 +23,15 @@ pub struct Settings {
     pub gpu_index: i64,
 }
 
+const DEFAULT_CONFIG_FILE_CONTENT: &str = r#"mine_to_address = ""
+rpc_url = "http://127.0.0.1:10604"
```

Comment:
Bight want to template this with the defaults?

Reply:
Isn't trivial on Rust, can do later